templates/partials/partial-overlay-image.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * FM Overlays Image Template
+ * used to create responsive picture element
+ */
+
+
+
+
+$image_sizes = Fm_Overlays_Helpers::instance()->get_overlay_image_sizes( $overlay->overlay_content['image_id'] );
+
+
+$classes = 'entry-thumbnail fm-image';
+$alt_text = the_title_attribute( array( 'echo' => false ) );
+
+?>
+
+
+<?php if ( ! empty( $image_sizes ) ) : ?>
+	<picture <?php if ( ! empty( $id ) ) : ?>
```

Comment:
My browsers still don't support `<picture>` (http://caniuse.com/#feat=picture). Does this plugin use [picturefill](https://github.com/scottjehl/picturefill) (if not, it should).